<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWidget</class>
 <widget class="QWidget" name="MainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>729</width>
    <height>586</height>
   </rect>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <widget class="KTabWidget" name="ktabwidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="automaticResizeTabs">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Assign user profiles</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QTreeView" name="userList">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="rootIsDecorated">
          <bool>false</bool>
         </property>
         <property name="uniformRowHeights">
          <bool>true</bool>
         </property>
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <property name="sizeConstraint">
          <enum>QLayout::SetDefaultConstraint</enum>
         </property>
         <item>
          <widget class="KPushButton" name="btnAssignUserProfile">
           <property name="text">
            <string>Assign profile</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="KPushButton" name="btnClearUserProfile">
           <property name="text">
            <string>Clear profile</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="KPushButton" name="btnDiscard1">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Discard</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="KPushButton" name="btnCommit1">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Save</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>../../../../../../../usr/share/icons/oxygen/16x16/status/dialog-password.png</normaloff>../../../../../../../usr/share/icons/oxygen/16x16/status/dialog-password.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Assign group profiles</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QTreeView" name="groupList">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="alternatingRowColors">
          <bool>false</bool>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <property name="indentation">
          <number>20</number>
         </property>
         <property name="rootIsDecorated">
          <bool>false</bool>
         </property>
         <property name="uniformRowHeights">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="KPushButton" name="btnAssignGroupProfile">
           <property name="text">
            <string>Assign profile</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="KPushButton" name="btnClearGroupProfile">
           <property name="text">
            <string>Clear profile</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="KPushButton" name="btnDiscard2">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Discard</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="KPushButton" name="btnCommit2">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Save</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>../../../../../../../usr/share/icons/oxygen/16x16/status/dialog-password.png</normaloff>../../../../../../../usr/share/icons/oxygen/16x16/status/dialog-password.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Manage profiles</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QTreeView" name="profileList">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="rootIsDecorated">
          <bool>false</bool>
         </property>
         <property name="uniformRowHeights">
          <bool>true</bool>
         </property>
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="KPushButton" name="btnNewProfile">
           <property name="text">
            <string>New</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="KPushButton" name="btnDeleteProfile">
           <property name="text">
            <string>Delete</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="KPushButton" name="btnSetupProfile">
           <property name="text">
            <string>Edit Profile</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="KPushButton" name="btnProfileProperties">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>Properties</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="KPushButton" name="btnDiscard3">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Discard</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="KPushButton" name="btnCommit3">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Save</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>../../../../../../../usr/share/icons/oxygen/16x16/status/dialog-password.png</normaloff>../../../../../../../usr/share/icons/oxygen/16x16/status/dialog-password.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KPushButton</class>
   <extends>QPushButton</extends>
   <header>kpushbutton.h</header>
  </customwidget>
  <customwidget>
   <class>KTabWidget</class>
   <extends>QTabWidget</extends>
   <header>ktabwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>ktabwidget</tabstop>
  <tabstop>userList</tabstop>
  <tabstop>btnAssignUserProfile</tabstop>
  <tabstop>btnClearUserProfile</tabstop>
  <tabstop>btnDiscard1</tabstop>
  <tabstop>btnCommit1</tabstop>
  <tabstop>groupList</tabstop>
  <tabstop>btnAssignGroupProfile</tabstop>
  <tabstop>btnClearGroupProfile</tabstop>
  <tabstop>btnDiscard2</tabstop>
  <tabstop>btnCommit2</tabstop>
  <tabstop>profileList</tabstop>
  <tabstop>btnNewProfile</tabstop>
  <tabstop>btnDeleteProfile</tabstop>
  <tabstop>btnSetupProfile</tabstop>
  <tabstop>btnProfileProperties</tabstop>
  <tabstop>btnDiscard3</tabstop>
  <tabstop>btnCommit3</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
